File: doc/sequentielle.java
public class Sequentielle {

    public static boolean contains(int[] a, int b){
        for (int i : a) {
            if (i==b){
                return true;
            }
        }
        return false;
    }
    public static void main(String[] args) {
        int[] a = {3,2,4,5,3,2,7,6,4};
        boolean ans = contains(a,7);
        if (ans) {
            System.out.println ("Nombre trouve");
        }
        else {
            System.out.println ("Nombre non trouve");
        }

    }
}

File: doc/sequentielle.py
""" -*- coding: utf-8 -*-
Auteur: Emmanuel Malan
date : 11-12-20  ;  langage : python
objet : recherche sequentielle """

def recherche_sequentielle(liste, element):
    a=0;
    b =len (liste)
    trouve = False
    i=a
    while ( trouve == False and i<b):
        if liste[i] == element :
            trouve = True
        i=i+1
    return trouve

# Programme principale pour tester le code ci-dessus
tab = [1, 22, 23, 74]
 
print (recherche_sequentielle(tab,15)) 
